TarSCM/scm/base.py

```diff
@@ -147,7 +147,9 @@ def prepare_clone_dir(self):
         # current work directory to allow the developer to work inside
         # of the git repo and fetch local changes
         if sys.argv[0].endswith("snapcraft") or \
-           (self.args.use_obs_scm and os.getenv('OSC_VERSION')):
+           (self.args.use_obs_scm and
+                os.getenv('OSC_VERSION') and
+                os.path.isdir('.git')):
```

Comment:
Do we want to support the case where the developer is in a subdirectory of the top-level working directory?

Reply:
I believe it does a change to the toplevel dir beforehand

Reply:
I'll believe you ;-)